### ISSUE TITLE
Bizarre geometry files created by isapt with FISAPT_DO_PLOT

### ISSUE THREAD

**Issue by lazaroid** [psi4/psi4#1954]
I've been fiddling with isapt with the FISAPT_DO_PLOT set to true to generate cube files, etc. The coordinates in the resultant geom.xyz files are very wrong, although the energies appear to be correct and all tests pass.

Running tests/isapt/input.dat through a conda nighty build (1.4a2.dev723) generates the following geom.xyz file (in both the plot and fsapt subdirectories):
```
19
0 1 C5H12O2
O                     0.000000000000     2.942220000000     0.000000000000
H                     0.000000000000     2.054360000000     0.000000000000
O                     0.000000000000     0.000000000000     0.000000000000
H                     0.000000000000     0.000000000000    -1.284510000000
C                     2.330480000000    -1.002690000000     0.000000000000
C                     1.897250000000     0.000000000000     0.000000000000
C                     2.282320000000     1.506690000000     0.000000000000
C                     1.822040000000     2.846080000000     0.000000000000
C                     2.379050000000     4.020990000000     0.000000000000
H                     3.412460000000    -1.030300000000     0.000000000000
H                     2.053620000000    -1.843720000000     0.000000000000
H                     1.827140000000    -1.163820000000     0.000000000000
H                     2.362430000000     0.000000000000    -1.576360000000
H                     3.369620000000     1.514140000000     0.000000000000
H                     1.812510000000     1.380600000000     1.281400000000
H                     2.143440000000     2.929670000000    -1.338430000000
H                     3.473200000000     4.024000000000     0.000000000000
H                     2.035350000000     3.992160000000     1.536350000000
H                     2.024810000000     4.967850000000     0.000000000000
```
which has lots of seemingly random (although repeatable) x, y, or z coordinates set to zero (and overlapping atoms) when compared with the expected geom.xyz from the plot-ref subdirectory:
```
19

 O    0.399870000000    2.942220000000   -0.265350000000
 H    0.058930000000    2.054360000000   -0.509620000000
 O    0.481220000000    0.302770000000   -0.777630000000
 H    0.261060000000   -0.500050000000   -1.284510000000
 C    2.330480000000   -1.002690000000    0.037710000000
 C    1.897250000000    0.315330000000   -0.590090000000
 C    2.282320000000    1.506690000000    0.297090000000
 C    1.822040000000    2.846080000000   -0.294320000000
 C    2.379050000000    4.020990000000    0.496390000000
 H    3.412460000000   -1.030300000000    0.198250000000
 H    2.053620000000   -1.843720000000   -0.607090000000
 H    1.827140000000   -1.163820000000    0.997340000000
 H    2.362430000000    0.423330000000   -1.576360000000
 H    3.369620000000    1.514140000000    0.438130000000
 H    1.812510000000    1.380600000000    1.281400000000
 H    2.143440000000    2.929670000000   -1.338430000000
 H    3.473200000000    4.024000000000    0.488190000000
 H    2.035350000000    3.992160000000    1.536350000000
 H    2.024810000000    4.967850000000    0.074550000000
'''

**Comment by JonathonMisiewicz** [comment]
"Nightly" isn't actually nightly, so it's missing some bug fixes. My hunch is that this is the same bug #1938 fixed. In which case, all we can do is tell you to compile from master or wait until we update "nightly" again. @bgpeyton, does this look right to you?

**Comment by lazaroid** [comment]
That's the fella! A quick edit of my psi4/driver/qcdb/molecule.py and the geometries now match.

:-)

Out of interest, what's the approximate delay between a pull request being merged and it appearing in a conda build? (Or is there any easy way to see whether something has been incorporated?) I was waiting for the TD-SCF commits to appear the other week!

**Comment by bgpeyton** [comment]
@JonathonMisiewicz you're correct. All coordinates below 1 were brought to zero. A quick update should fix it, or you may be able to wrangle the output of `mol.print_out()` to suit your needs.

**Comment by JonathonMisiewicz** [comment]
I don't have an approximate delay.

Nightly gets updated whenever @loriab updates it. There's no deeper pattern, that I'm aware of.

**Comment by JonathonMisiewicz** [comment]
This bug was fixed in #1938, so closing.
